media/static-bundles.json

```diff
@@ -867,6 +853,14 @@
       ],
       "name": "firefox_accounts"
     },
+    {
+      "files": [
+        "css/base/mozilla-fxa-state.scss",
+        "css/base/mozilla-fxa-form.scss",
```

Comment:
We should be careful when importing Pebbles CSS files into the same bundle as Protocol. This one imports the pebbles `/pebbles/includes/lib/` and `/pebbles/components/buttons/`. Both can have adverse side effects, such as overriding mixins, and in this case the download buttons that should be blue are showing up as green (which is something specific to non-protocol pages).

If you need these styles, I would create a Protcol equivalent and [put it here](https://github.com/mozilla/bedrock/tree/master/media/css/protocol/components)

Reply:
We've added a Protocol-flavored form style